.env.example

```diff
@@ -8,4 +8,6 @@ GITHUB_SECRET=
 GOOGLE_ID=
```

Comment:
renamed the example env file based on these examples


https://github.com/vercel/platforms/blob/main/.env.example


https://github.com/vercel/ai/blob/main/examples/next-openai/.env.local.example